src/main/java/com/singh/astha/notificationservice/config/AppConfig.java

```diff
@@ -0,0 +1,19 @@
+package com.singh.astha.notificationservice.config;
+
+
+import io.swagger.v3.oas.annotations.OpenAPIDefinition;
+import io.swagger.v3.oas.annotations.enums.SecuritySchemeType;
+import io.swagger.v3.oas.annotations.info.Info;
+import io.swagger.v3.oas.annotations.security.SecurityScheme;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@OpenAPIDefinition(info = @Info(title = "My API", version = "v1"))
+@SecurityScheme(
+        name = "bearerAuth",
+        type = SecuritySchemeType.HTTP,
+        bearerFormat = "JWT",
+        scheme = "bearer"
+)
+public class AppConfig {
```

Comment:
nit: If this file is just for the Swagger Docs configuration, then we can name it OpenAPIConfig

Reply:
fixed